etc/config/c++.win32.properties

```diff
@@ -1 +1,38 @@
-supportsBinary=true
+# Default settings for C++ on Windows
+
+compilers=&vc2017:&clang
+postProcess=
+demanglerClassFile=./demangler-win32
+objdumper=objdump
+options=
+supportsBinary=false
+stubRe=\bmain\b
+stubText=int main(void){return 0;/*stub provided by Compiler Explorer*/}
+
+
+# clang compilers
+
+group.clang.compilers=&clang_32:&clang_64
+group.clang.intelAsm=-mllvm --x86-asm-syntax=intel
+group.clang.groupName=clang
+group.clang.compilerType=win32
+group.clang.binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|\.plt.*)$
+
+group.clang_32.groupName=clang x86
+group.clang_32.options=-m32
+group.clang_64.groupName=clang amd64
+
+
+# visual C++ 2017 compilers
+
+group.vc2017.compilers=&vc2017_32:&vc2017_64
+group.vc2017.options=-EHsc
+group.vc2017.compilerType=win32-vc
+group.vc2017.needsMulti=false
+group.vc2017.includeFlag=/I
+group.vc2017.versionFlag=/?
+group.vc2017.versionRe=^Microsoft \(R\) C/C\+\+.*$
```

Comment:
Minor problem: Non english cl.exe will probably not match this regex. Optionally, it would be great if we could find some other pattern for this, but we won't delay the merge just for this, so don't worry too much :)

Reply:
C'est un bon point!

Reply:
Yeah I have no idea how I would do this lol

Reply:
how about just "Microsoft \(R\).*$" ?

Reply:
What about languages where Microsoft is localized?

Reply:
they should download proper version of VS

Reply:
It's all one version of VS, just different language packs. But, for example, this is the Chinese version line:

```
用于 ARM64 的 Microsoft (R) C/C++ 优化编译器 19.15.26720 版
```

What do we use this for anyways?